debezium-e2e-benchmark/py/tpc-run-test.py

```diff
@@ -0,0 +1,223 @@
+        print("%s:%d:%d: key=%s value=%s" % (message.topic, message.partition,
+                                             message.offset, message.key,
+                                             message.value))
+
+        d = json.loads(message.value)
+        file.write(str(message.timestamp) + "000;" + str(d['payload']['after']['T0']) +
+                   ";" + d['payload']['op'] + ";" + str(i) + ";" + str(d['payload']['after']['ID']) + "\n")
+        i = i + 1
+        print(i)
+        print(count)
+        if i >= count:
+            break
+    file.close()
+
+
+def getjdbcconnection(config, tpcconfig):
+    connectiontype = config['config']['connector.class'].split('.')[3]
+    jdbctype = 'jdbc:' + connectiontype + '://'
+    if connectiontype == 'oracle':
+        jdbctype = 'jdbc:oracle:thin:@'
+    conn = jaydebeapi.connect(tpcconfig['jdbc'][connectiontype]['jdbcdriver'], jdbctype +
+                              config['config']['database.hostname'] + ':' +
+                              config['config']['database.port'] + '/' +
+                              config['config']['database.dbname'],
+                              [config['config']['database.user'],
+                                  config['config']['database.password']],
+                              '/home/tpc/jdbcdriver/' + tpcconfig['jdbc'][connectiontype]['jar'])
+    return conn
+
+
+def main(argv):
+    with open('../register.json') as f:
+        config = json.load(f)
+    with open('tpc-config.json') as f:
+        tpcconfig = json.load(f)
+
+    print(config['config']['connector.class'])
+    print(config['name'])
+    config['name'] = 'tpc-connector'
+    print(config['name'])
+    config['config']['database.whitelist'] = 'TPC.TEST'
+
+    config['config']['database.history.kafka.topic'] = 'tpc-test'
+
+    databasetype = config['config']['connector.class']
+    print(databasetype.split('.')[3])
+    print(databasetype)
+    conn = getjdbcconnection(config, tpcconfig)
+
+    initsql(conn, config, tpcconfig)
+    createTPCTable(conn, config, tpcconfig)
+    enablecdctablesql(conn, config, tpcconfig)
+
+    print('============')
+
+    print('get status tpc connector')
+    resp = requests.get(
+        'http://' + tpcconfig['debezium.connect.server'] + '/connectors/tpc-connector/status',  verify=False)
+    print(resp.content)
+    print(resp.status_code)
+    retvalue = json.loads(resp.content)
+    print(retvalue)
+
+    print('============')
+
+    resp = requests.delete(
+        'http://' + tpcconfig['debezium.connect.server'] + '/connectors/tpc-connector',  verify=False)
+    print(resp.content)
+    print(resp.status_code)
+    # retvalue = json.loads(resp.content)
+    # print(retvalue)
+    if (resp.status_code == 404):
+        print('tpc-connector not exists')
+    else:
+        print('tpc-connector deleted')
+        pass
+
+
+    dockerbootstrapserver = config['config']['database.history.kafka.bootstrap.servers']
```

Comment:
Needs to be passed via CLI opts for databases not using database history

Reply:
I read in this way the bootstrap.servers. What else is the correct config param from register.json ?